scripts/correlateTBN.py

```diff
@@ -200,7 +201,7 @@ def main(args):
     print("==")
     print("Station: %s" % station.name)
     print("Date observed: %s" % date)
-    print("Julian day: %.5f" % jd)
+    print("Julian day: %.5f" % mjd)
```

Comment:
I really did mean JD and not MJD.  The JD should now be part of the `lsl.reader.base.FrameTimestamp` class.

Reply:
Oh I see. It's not in the release. In master `lsl.reader.base.FrameTimestamp` does include jd but I just did a fresh install of the release and there is only an mjd.

The release was installed in a python3.6 virtual environment with all the dependencies met. lsl was installed with `pip install lsl` and `pip freeze` lists the the lsl version as 2.0.0.

<venv>/lsl2-release/lib/python3.6/site-packages/lsl/reader/base.py has the following `@property` definitions inside the `FrameTimeStamp` class:

unix, mjd, pulsar_mjd, dp_timetag, datetime, astropy.


The release correlateTBN.py script also has some python2 syntaxed exceptions `except Exception, e:` which don't look like they'll exist on master.

I'm now setting up a venv with py3.6 in which I'll install lsl from master. That should fix these problems.